ner/pom.xml

```diff
@@ -61,7 +61,7 @@
         <dependency>
             <groupId>edu.illinois.cs.cogcomp</groupId>
             <artifactId>LBJava</artifactId>
-            <version>1.2.26</version>
+            <version>1.3.0</version>
```

Comment:
Are we redeploying the models too?

Reply:
I would really like to do a parameter sweep on ontonotes, more out of curiosity than anything driving need to further improve it. Perhaps we should wait and see if I do that before we rebuild the models? I suspect the sweep on ontonotes only would only take a week or so at the most.